Packs/ThreatConnect/Integrations/ThreatConnect_v2/ThreatConnect_v2.yml

```diff
@@ -507,6 +514,24 @@ script:
     - contextPath: TC.Indicator.File.SHA256
       description: The SHA256 hash of the indicator of the file.
       type: string
+    - contextPath: TC.Indicator.IndicatorAttribute.dateAdded
```

Comment:
Please note that in the code you output to `IndicatorAttributes` so I suppose this is a typo?

Reply:
That was a typo, it has been corrected.